sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -53,28 +80,39 @@ typedef struct
   az_span option;
 } az_storage_blobs_blob_upload_options;
 
+/**
+ * @brief Init a client with default options
```

Comment:
This method doesn't initialize a client, but rather just creates the default options that can be passed into the init client method.

```suggestion
 * @brief Gets the default blob storage options.
```